app/Http/Controllers/HardwareController.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\NethserverHardware;
+use App\Models\NethsecurityHardware;
+use App\Http\Controllers\SelectController;
+use Illuminate\Http\Request;
+
+class HardwareController extends Controller
+{
+    public function index(Request $request, SelectController $selectController)
```

Comment:
Instead of passing a `SelectController` (not recommended), after editing the `routes/web.php`, you can add a `string $installation` to the parameters, use it to know which system you wanna search the system about (I.E. `Nethserver` or `Nethsecurity`)